tests/events/main.c

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Please add a readme and/or a `pexpect` script.

Reply:
done